libs/openFrameworks/utils/ofFileUtils.cpp

```diff
@@ -1695,24 +1691,23 @@ std::string ofFilePath::addTrailingSlash(const of::filesystem::path & _path){
 			path = (path / sep);
 		}
 	}
-//	return path;
 	return path.string();
 #endif
 }
 
 
 //------------------------------------------------------------------------------------------------------------
+// FIXME: - deprecate and suggest using of::filesystem::path.extension() instead
 string ofFilePath::getFileExt(const of::filesystem::path& filename){
```

Comment:
Yes agreed, removed the comment. 
I've noticed api names are different ofFilePath::getFileExt and ofFile::getExtension
it would be great to deprecate over time things that are covered by std::filesystem, so there are less things to mantain and to document too :)